src/post/post.service.integration.test.ts

```diff
@@ -203,7 +203,7 @@ describe('PostService', () => {
 
             // Verify mention is wrapped in hyperlink in content
             expect(post.content).toBe(
-                `<p>This is a test note mentioning <a href="${mentionedAccount.apId}" rel="nofollow noopener noreferrer">@${mentionedAccount.username}@${mentionedAccount.apId.hostname}</a></p>`,
+                `<p>This is a test note mentioning <a href="${mentionedAccount.apId}" data-profile="@${mentionedAccount.username}@${mentionedAccount.apId.hostname}" rel="nofollow noopener noreferrer">@${mentionedAccount.username}@${mentionedAccount.apId.hostname}</a></p>`,
```

Comment:
We can also use `mentionedAccount.domain` now

Reply:
This is a nice suggestion sag but the tests are using [this function](https://github.com/TryGhost/ActivityPub/blob/main/src/account/account.repository.knex.ts#L202) to get the account and domain is not selected here. So I am gonna leave it like this for now.